src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -16,7 +17,7 @@ import scala.util.Try
 
 object Backend {
   lazy val LocalBackend = new LocalBackend
-  lazy val JesBackend = new JesBackend { JesConf } //forces configuration resolution to fail now if something is missing
+  lazy val JesBackend = new JesBackend with JesAuthenticationSingleton { JesConf } //forces configuration resolution to fail now if something is missing
```

Comment:
Actually yes that could work too, I think I had a reason for doing that at the time but I can't remember why, and I just tried your way which works and is less verbose. I'll make sure everything still works this way and change to that if it does.